core/errors.go

```diff
@@ -79,3 +79,6 @@ var ErrInvalidLogFileMinLifeSpan = errors.New("minimum log file life span is inv
 
 // ErrFileLoggingProcessIsClosed signals that the file logging process is closed
 var ErrFileLoggingProcessIsClosed = errors.New("file logging process is closed")
+
+// ErrInvalidTransactionVersion signals  that an invalid transaction version has been provided
```

Comment:
remove an empty space between `signals` and `that`

Reply:
done.